firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/I2C/HoneywellABP/Sensor.h

```diff
@@ -37,15 +37,19 @@ class Sensor : public Initializable {
   InitializableState output(float &output);
 
  private:
+  using Action = StateMachine::Action;
+
   static const size_t max_retries_setup = 8;    // max retries for setup
   static const size_t max_retries_measure = 8;  // max retries for measuring
 
+  const float pmin = 0.0;  // minimum pressure for abpxxxx001pg2a3
+  const float pmax = 1.0;  // maximum pressure for abpxxxx001pg2a3
```

Comment:
We should probably rename these variables `p_min` and `p_max` instead of `pmin` and `pmax`.

Anywhere we have numeric literals for physical quantities, we should also annotate them with units if the units are known, e.g.:

```
  const float p_min = 0.0;  // psi; minimum pressure for abpxxxx001pg2a3
  const float p_max = 1.0;  // psi; maximum pressure for abpxxxx001pg2a3
```

(assuming the abpxxxx001pg2a3 uses PSI)